workflows/repo-specific/cloudquery/policy_test.yml

```
@@ -0,0 +1,115 @@
+name: test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [main]
+
+env:
+  CGO_ENABLED: 0
+  CQ_NO_TELEMETRY: 1
+
+jobs:
+  unitests:
+    strategy:
+      matrix:
+        dbversion: ['postgres:latest']
+        go: ['1.17']
+        platform: [ ubuntu-latest]
```

What do you think about using a matrix to run this in parallel?

```suggestion
        dbversion: ['postgres:latest']
        go: ['1.17']
        platform: [ ubuntu-latest]
        provider: ['aws', 'gcp', 'azure', 'k8s']
```

Then `go run ./main.go provider build-schema ${{ matrix.provider }}` etc.

This works since the provider name matches the policy name, but we can also use a tuple ([provider, policy]) in the future if needed